packages/hardhat/package.json

```diff
@@ -5,6 +5,7 @@
   "license": "MIT",
   "devDependencies": {
     "@typescript-eslint/parser": "^4.29.1",
+    "@uniswap/v3-sdk": "^3.8.2",
```

Comment:
why we need this ?

Reply:
I think this was me trying something with getting quotes but not needed now.

Reply:
I think @alpinechicken  added this, I can remove it.